src/middlewares/requestLogger.ts

```diff
@@ -0,0 +1,87 @@
+import { Request, Response, NextFunction } from 'express'
+import { CONFIG as config } from '../config'
+import createLogger from '../utils/logger'
+import crypto from 'crypto'
+
+// default to true if not set
+const enableConsole = process.env.SHARDEUM_JSONRPC_CONSOLE_LOGGING === 'true'
+
+let enableFile: boolean = true;
+if (process.env.SHARDEUM_JSONRPC_FILE_LOGGING === 'true') {
+  enableFile = true;
+} else if (process.env.SHARDEUM_JSONRPC_FILE_LOGGING === 'false') {
+  enableFile = false;
+} 
+
+const logger = createLogger({
+  enableConsole,
+  enableFile,
+  filename: 'logs/requests.log',
```

Comment:
I wonder if we need to make any changes to ldrpc-docker to be able to mount these logs

Reply:
yea we will need to share this `logs/` directory as some named docker volume like `-v request_logs:/home/node/json-rpc-server/logs/`

Reply:
I think we should mount them to same folder(named volume) as other logs. Is it possible to drop this log file in ~/.pm2/logs folder instead of logs folder?
Also, is there any rotation that happens for this log file?

Reply:
for log rotation `pino` recommends to just use linux's `logrotate` in their docs:


https://github.com/pinojs/pino/blob/1934cda1be0717714631584e3cbfa6fb0ef4224b/docs/help.md#rotate

the host can run it or we can run it in ldrpc if we need to.

Reply:
> I think we should mount them to same folder(named volume) as other logs. Is it possible to drop this log file in ~/.pm2/logs folder instead of logs folder?

ill add a logs dir env variable so we can set it to whatever we need like that pm2 logs dir in docker container